docs/README.md

```diff
@@ -207,7 +207,10 @@ The site theme/template lives in `themes/thxvscode`.
 | `build:api-documenter:default` | --- |
 | `build:api-documenter:win32` | --- |
 | `build:diagrams` | Generate the diagram images using ditaa. |
+| `build:fast` | Builds the site in a fast, but incomplete way. Useful for testing and interation. |
```

Comment:
build:fast does complete builds. Really the only difference is that is uses Curtis' custom build tool. The only thing with build:fast is that it's not the way we build in the CI. I've voted for a while to just have build:fast be the only build but I'm sure there's a reason why we haven't @curtisman ?

```suggestion
| `build:fast` | Alternate build solution for fast incremental builds. Useful for local building and iteration. |
```

Reply:
This is the package.json file in the docs folder, not the root. I.e. this isn't fluid build. 😃